plugins/autosuspend.koplugin/main.lua

```diff
@@ -83,11 +84,11 @@ function AutoSuspend:_schedule()
         UIManager:suspend()
     else
         if self:_enabled() then
-            logger.dbg("AutoSuspend: schedule suspend at ", os.time() + delay_suspend)
+            logger.dbg("AutoSuspend: schedule suspend in", delay_suspend)
```

Comment:
Logger entries are already timestamped, and the scheduled action itself will also trigger a log entry, so it feels like simply loading the actual scheduleIn delay is actually more useful?